portal/src/main/webapp/WEB-INF/jsp/visualize.jsp

```diff
@@ -85,47 +85,52 @@
 
                 if (values != null && values.length >0)
                 {
-                    for (int i=0; i<values.length; i++)
-                    {
+                    for (int i=0; i<values.length; i++) {
+                    	
+                    	Boolean genesetQuery = false;
+		               	if (localGeneSetList.length() > 0 && localGeneList.length() == 0) {
+                    		genesetQuery = true;
+                    	}
+                    	
                         String currentValue = values[i].trim();
 
-                        if (currentValue.contains("mutation") && !disabledTabs.contains("mutations"))
+                        if ((currentValue.contains("mutation") && !disabledTabs.contains("mutations")) && !genesetQuery)
```

Comment:
I totally see your point. But the sole purpose of `disabledTabs` here is to control which tab to show.  Instead of putting `genesetQuery` to each if-else condition I would prefer to do something like this
```
if(genesetQuery) {
    var array = ['IGV', ...];
    disabledTabs = _.union(disabledTabs, array);
}
```
What do you think?